README.md

```diff
@@ -82,7 +82,7 @@ in PEM format (e.g. /opt/venafi/bundle.pem) and include it using the
 If you are using Venafi as a Service, verify the following:
 
 - The Venafi as a Service REST API at [https://api.venafi.cloud](https://api.venafi.cloud/swagger-ui.html)
-is accessible from the system where Terraform will run.
+or https://api.eu.venafi.cloud(if you have an EU account) is accessible from the system where Terraform will run.
```

Comment:
need to format the url in the same way as the other one:
```
[https://api.eu.venafi.cloud](https://api.eu.venafi.cloud/swagger-ui.html)
```

Reply:
Done